packages/node-experimental/src/integrations/graphql.ts

```diff
@@ -32,9 +32,7 @@ export class GraphQL extends NodePerformanceIntegration<void> implements Integra
       new GraphQLInstrumentation({
         ignoreTrivialResolveSpans: true,
         responseHook(span) {
-          addOtelSpanData(span.spanContext().spanId, {
-            origin: 'auto.graphql.otel-graphql',
-          });
+          addOriginToOtelSpan(span, 'auto.graphql.otel.graphql');
```

Comment:
also noticed some of these have still been incorrectly using `-` notation, which does not work.